docs/modules/ROOT/pages/tmail-backend/webadmin.adoc

```diff
@@ -576,3 +576,59 @@ Valid status includes:
 - `IN_PROGRESS`: Currently executed
 - `FAILED`: Error encountered while executing this step. Check the logs.
 - `ABORTED`: Won't be executed because of previous step failures.
+
+=== Delete data of a user
```

Comment:
IMO not needed: we already documented this endpoint on James side

Reply:
Not even saying there is an extra step? Can at least point to the james doc and say there is an extra step to it with tmail?

Reply:
rollbacked the docs. A bit redundant is better than missing IMO...